website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -23,15 +22,27 @@ part of the process to [Integrating Kubebuilder and Operator SDK][integration-do
 
 ### What is new
 
-Scaffolded projects now use:
+Projects are now scaffold using:
 
 - [kustomize][kustomize] to manage Kubernetes resources needed to deploy your operator
 - A `Makefile` with helpful targets for build, test, and deployment, and to give you flexibility to tailor things to your project's needs
 - Updated metrics configuration using [kube-auth-proxy][kube-auth-proxy], a `--metrics-addr` flag, and [kustomize][kustomize]-based deployment of a Kubernetes `Service` and prometheus operator `ServiceMonitor`
+- A preliminary support for plugins. For more info see the [extensible CLI and scaffolding plugins design document][plugins-phase1-design-doc]
```

Comment:
```suggestion
- [CLI plugins][plugins-phase1-design-doc] for those interested in maximum project customization.
```

Reply:
This text is == kb which is the silly suggestion for that which imo shows more accurate. 
Before 1.5 and 2 plugins impl can we really provide `maximum project customization`?